c2corg_ui/static/js/documentediting.js

```diff
@@ -33,7 +33,14 @@ app.documentEditingDirective = function() {
     scope: true,
     controller: 'appDocumentEditingController',
     controllerAs: 'editCtrl',
-    bindToController: true
+    bindToController: true,
+    link: function(scope, el, attrs, ctrl) {
+      // The directive executes after the dom is loaded but we have to wait until the expressions are evaluated
```

Comment:
I thought about it but it implies to specify a number of steps for every waypoint type and it's some lines of code more...or add this propriety to the DB? i don't know.